arrow/arrow.py

```diff
@@ -1057,6 +1057,79 @@ def humanize(
                 )
             )
 
+    def dehumanize(self, timestring):
+        """ Returns an arrow object relative to the humanized difference in time given.
+            Only works with specifically formatted strings for the time being.
+
+        :param timestring: A string in the format returned by humanize by default
+            Number must be before unit of time, i.e. (4 hours)
+
+        Usage:
+
+            >>> arrow.utcnow()
+            <Arrow [2020-04-17T00:57:03+00:00]>
+
+            >>> arrow.utcnow().dehumanize("4 hours ago")
+            <Arrow [2020-04-17T20:57:03+00:00]>
```

Comment:
It would be nice if this didn't require an existing object kind of like how [humanize](https://github.com/crsmithdev/arrow/blob/master/arrow/arrow.py#L866) works currently, i.e.

```shell
>>> arrow.dehumanize("in 4 hours")
<Arrow [2020-04-19T23:58:23+00:00]>
```
This provides flexibility while allowing the user to select a different "now" if required.

Reply:
Like making dehumanize a class function instead?